src/utils/urls.js

```diff
@@ -144,18 +145,12 @@ function getRoomUrl(roomId) {
   return concatParts(roomsPrefix, encodeURIComponent(roomId));
 }
 
+function getLessonUrl(id, slug) {
+  return concatParts(lessonsPrefix, encodeURIComponent(id), encodeURIComponent(slug));
```

Comment:
`encodeURIComponent(slug)` will escape slashes in the slug, therefore altering the path. I see that we use them "literally" in the somewhat analogous case of docs (which is also not good). I think, to be 100% correct, we would have to do in both cases:

~~~
slug.split('/').map(x => encodeURIComponent(x)).join('/')
~~~
?

Reply:
Good point, I've used this for both lessons and documents, thanks!